.github/sync/config.yml

```
@@ -98,3 +95,30 @@ group:
       # release docs
       - source: .github/sync/RELEASE.md
         dest: RELEASE.md
+group:
+# Code of Conduct sync
+  - repos: |
```

We just need to append it to the existing group:

```suggestion
  # Code of Conduct sync
  - repos: |
```